tla/consensus/ccfraft.tla

```diff
@@ -959,7 +959,7 @@ AppendEntriesAlreadyDone(i, j, index, m) ==
     /\ Reply([type           |-> AppendEntriesResponse,
               term           |-> currentTerm[i],
               success        |-> TRUE,
-              lastLogIndex   |-> m.prevLogIndex + Len(m.entries),
+              lastLogIndex   |-> Len(log[i]),
```

Comment:
Adding 2 TLA changes to this PR, so that trace validation continues to work with the new rollback point.

This is aligning with implementation - an AER ACK always lists the node's `state->last_idx`. It does not consider the incoming message.

Reply:
Simulation has found a violation of `LogInv` that I believe is down to this. Investigating further.

Reply:
Linking in the other direction for posterity - this change is incorrect, and we need to align the impl with the spec. That's done in #6001.